Automattic/wp-calypso
# Reorganize constants in calypso-products package

### Changes proposed in this Pull Request

This PR reorganizes the constants of the `calypso-products` package into multiple files, for readability.

### Testing instructions

- Review code (checking the `constants` folder in your editor may be easier than checking the diff)
- Check that all tests pass
- Make sure you can boot up Calypso and cloud successfully
